proxyprefix/wsgi.py

```diff
@@ -0,0 +1,13 @@
+class ReverseProxiedApp(object):
+    """WSGI middleware to prefix script name with X-Forwarded-Prefix header."""
+
+    def __init__(self, app):
+        self.app = app
```

Comment:
I only mentioned it because in the mod_wsgi example docs, it was private. I tend to decide whether or not something should be public based on whether or not I am actively anticipating it being used externally, which likely won't ever be the case here. I don't feel strongly about it either.